### PR TITLE
doc: Add step-by-step manual mod-install example

### PR DESCRIPTION
**Pull request by x10an14**
Add an _OS-agnostic_ **_and_** _Client-agnostic_ (ref: Steam vs Epic Games Launcher) how-to example of how to enable mods.
Motivation for this is to clarify the instructions already existing on the page - especially for those of us unable to leverage SMM in its current state (Linux + Steam).

PS: Added a disclaimer to allay any fears that others could take this as a reason for hounding this community for support in the future.

### CODE REVIEW

**Comment by Panakotta00** [comment]
Please change your request target to the development branch, we don't merge PRs to master directly.
Else I would approve it ^^

**Comment by x10an14** [comment]
> Please change your request target to the development branch, we don't merge PRs to master directly.
> Else I would approve it ^^

Much appreciated @Panakotta00! I assumed you meant the `Dev` branch, which is now the target!

File: modules/ROOT/pages/ManualInstallDirections.adoc
Comment (Vilsol) [inline modules/ROOT/pages/ManualInstallDirections.adoc:81]: Please change this to use a `latest` url (https://github.com/satisfactorymodding/SatisfactoryModBootstrapper/releases/latest)

File: modules/ROOT/pages/ManualInstallDirections.adoc
Comment (Vilsol) [inline modules/ROOT/pages/ManualInstallDirections.adoc:85]: Please change this to use a `latest` url (https://github.com/satisfactorymodding/SatisfactoryModLoader/releases/latest)

**Review by Vilsol — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/ManualInstallDirections.adoc
Comment (x10an14) [reply modules/ROOT/pages/ManualInstallDirections.adoc:81]: Is it preferable to ensure backwards compatibility (keep it as is), or add the responsibility to update/correct documentation if anything ever changes?

Would be nice to hear from maintainers (you maybe?) where you'd like to draw that line before I change it =)
Don't feel strongly about it myself, would like to give the choice to the majority of the maintainers.

Comment (x10an14) [reply modules/ROOT/pages/ManualInstallDirections.adoc:81]: @Vilsol ---^

Comment (budak7273) [reply modules/ROOT/pages/ManualInstallDirections.adoc:81]: The docs aren't always updated when new SML updates release, so I would rather have it as a `latest` link. That way we won't have to update it with every release. If there's a problem and a certain old release has to be used, we would ideally put a note about that here.

Comment (x10an14) [reply modules/ROOT/pages/ManualInstallDirections.adoc:81]: Aight, that's two for, and no against in as many days! Change incoming as soon as I get a chance!

Comment (x10an14) [reply modules/ROOT/pages/ManualInstallDirections.adoc]: Done! =)

File: modules/ROOT/pages/ManualInstallDirections.adoc
Comment (x10an14) [reply modules/ROOT/pages/ManualInstallDirections.adoc]: Done! =)

**Comment by x10an14** [comment]
Done @Vilsol =)

File: modules/ROOT/pages/ManualInstallDirections.adoc
Comment (Vilsol) [inline modules/ROOT/pages/ManualInstallDirections.adoc:94]: This reference is not referenced anywhere

**Review by Vilsol — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/ManualInstallDirections.adoc
Comment (x10an14) [reply modules/ROOT/pages/ManualInstallDirections.adoc:94]: Thanks for catching that blunder! Fix'd =)

**Review by Vilsol — APPROVED** [review]

**Comment by budak7273** [comment]
Thanks!

**Pull request merged**
